pkgs/development/embedded/avrdude/default.nix

```diff
@@ -12,16 +12,16 @@ in
 
 stdenv.mkDerivation (finalAttrs: {
   pname = "avrdude";
-  version = "7.3";
+  version = "8.0";
 
   src = fetchFromGitHub {
     owner = "avrdudes";
-    repo = "avdude";
+    repo = "avrdude";
     rev = "v${finalAttrs.version}";
-    sha256 = "sha256-JqW3AOMmAfcy+PQRcqviWlxA6GoMSEfzIFt1pRYY7Dw=";
+    sha256 = "w58HVCvKuWpGJwllupbj7ndeq4iE9LPs/IjFSUN0DOU=";
```

Comment:
Please use an SRI hash here:

```
hash = "sha256-w58HVCvKuWpGJwllupbj7ndeq4iE9LPs/IjFSUN0DOU=";
```

Reply:
Will change this at the beginning of next week as I am on holiday right now.

Reply:
Ok. I plan to merge tonight, as I think sha256 vs hash is orthogonal to the version bump.